modules/rhizo_base/manifests/init.pp

```diff
@@ -256,6 +256,8 @@
   file { '/home/rhizomatica/bin/vars.sh':
       ensure  => present,
       content => template('rhizo_base/vars.sh.erb'),
+      owner   => rhizomatica,
```

Comment:
Why? I think this was intentional. Better might be to move these scripts elsewhere.

Reply:
Okay, that's reasonable! Is there a usual location where root own scripts are installed in rccn?